frontend/components/buttons/Button/Button.tsx

```diff
@@ -39,6 +38,8 @@ export interface IButtonProps {
       ) => void);
   isLoading?: boolean;
   customOnKeyDown?: (e: React.KeyboardEvent) => void;
+  /** Required for buttons that contain SVG icons using`stroke` instead of`fill` for proper hover styling */
+  iconStroke?: boolean;
```

Comment:
No more one of styling to fix this pesky fill vs. stroke bug, cc: @jacobshandling you can use this property for your play button hover issue

Reply:
Just what the doctor ordered, thanks Rachel!

<img width="504" alt="Screenshot 2025-04-23 at 2 54 10 PM" src="https://github.com/user-attachments/assets/c4b598c5-7d91-43c5-a9e0-5af70e133d41" />